t/cli/cli.t

```diff
@@ -58,3 +58,21 @@ nameserver 172.27.0.1
 nameserver fe80::215:5dff:fec5:8e1d
 --- response_body
 ["172.27.0.1","fe80::215:5dff:fec5:8e1d"]
+
+
+
+=== TEST 2: It is forbidden to run APISIX under the /root directory(is_root_path is true).
```

Comment:
we should use `shell` script to test this case

Reply:
updated.